icu4c/source/i18n/unitconverter.cpp

```diff
@@ -82,6 +82,42 @@ class UnitConversionRatesSink : public ResourceSink {
     ConvertUnit *convertUnit;
 };
 
+/*
+ Converts `UnicodeString` to a DecNum.
```

Comment:
This function converts a StringPiece to a DecNum, not a UnicodeString to a DecNum.